site/_posts/2015-01-20-jekyll-meet-and-greet.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: news_item
+title: "Jekyll Meet & Greet at GitHub HQ"
+date: "2015-01-20 19:23:12 -0800"
+author: parkr
+categories: [meetup]
+---
+
+Hey! Our friends at GitHub have agreed to host a Jekyll meet & greet on
+**February 5, 2015 at 7pm**. The event will be hosted at [GitHub's
+Headquarters](https://www.google.com/maps/place/GitHub/@37.782084,-122.391403)
```

Comment:
We should switch this to point to 85 Colin P. Kelley, which is the visitor entrance (people will need to come through here anyway): http://goo.gl/qMeBi5